### PR TITLE
AO3-4291 Screen readers do not read help text on forms

### PR DESCRIPTION
**Pull request by sarken**
https://otwarchive.atlassian.net/browse/AO3-4291

Was https://code.google.com/p/otwarchive/issues/detail?id=4303

Makes it so our help text on a bunch of forms is automatically read by screen readers by [using aria-describedby](http://www.paciellogroup.com/blog/2014/12/using-aria-describedby-to-provide-helpful-form-hints/).

Unfortunately, the version of Ruby we're on seems to require the hash rocket syntax for this HTML attribute, but I did a bunch of hash rocket to json style conversion to try to make up for it. I also changed the field labels from title case with colons ("New User Name:") to our new sentence-case, colon-less standard ("New user name").

### CODE REVIEW

File: app/views/works/_standard_form.html.erb [hunk 1]
```diff
@@ -1,40 +1,43 @@
 <!--REVIEW: trying out articles on these fieldsets to see if that's a better chunker, but may revert after testing. Have also tossed in landmark headings on top level groups-->
```
Comment (ariana-paris) [inline app/views/works/_standard_form.html.erb]: Not directly related to the issue at hand, but is this comment still relevant?

Comment (sarken) [inline app/views/works/_standard_form.html.erb]: Partially. I'm conflicted about the article roles and what to do about them, but the landmark headings are standard operating procedure for us at this point.

**Comment by sarken** [comment]
Ugh, have to update, marking Needs Coder Action.

**Comment by sarken** [comment]
Updated. Twas just a conflict in a test.

**Pull request merged**